pages/FAQ/_index.md

```diff
@@ -328,3 +332,7 @@ If the pop-up disappears immediately, you can use:
 ```ini
 windowrulev2 = minsize 1 1, title:^(TITLE)$, class:^(CLASS)$
 ```
+
+# Steam's file picker no worky
+
+On instances where you have a steam library on another drive that you have to add, the Hyprland's file picker would not normally appear when selecting a directory from steam. Steam have its own file picker, however, it's not functional. Install `xdg-desktop-portal-gtk` to show the desktop's file picker.
```

Comment:
Steam _has_ its own file picker

Reply:
Yes, steam does have its own file picker. However, their file picker does not work. The file picker window opens but none of the buttons work and you can't select files. The image below is without xdg-desktop-portal-gtk.
![swappy-20240212_005252](https://github.com/hyprwm/hyprland-wiki/assets/103093878/a4c0a1b5-9e1f-4a0a-8a98-4a8b69fe9c57)

Reply:
I suggest you split the message into multiple lines.

Reply:
my problem was with the grammar not the content
`have` -> `has`

Reply:
Alright, I have made additional commits.